deploy/default.conf

```diff
@@ -110,7 +110,7 @@ http {
     add_header Strict-Transport-Security "max-age=31536000";
     add_header X-Frame-Options "SAMEORIGIN" always;
 
-    client_max_body_size 200m;   # adjust to taste
+    client_max_body_size 350m;   # adjust to taste
```

Comment:
To match https://github.com/dsaidgovsg/ae-app/blob/af023d9b566717f8191d73d429aef5d919939d64/charts/preprocess/templates/_helpers.tpl#L73 and provide some additional buffer in case file size increases due to additional first description column.